includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -78,9 +78,26 @@ protected function render() {
 
 		$this->adsense_tag_printed = true;
 
+		$amp_auto_ads_opt = array(
+			'data-ad-client' => $this->tag_id,
+		);
+
+		$amp_auto_ads_opt_filtered = apply_filters( 'googlesitekit_amp_auto_ads_attributes', $amp_auto_ads_opt );
+
+		if ( ! is_array( $amp_auto_ads_opt_filtered ) ) {
+			$amp_auto_ads_opt_filtered = $amp_auto_ads_opts;
+		}
+
+		$amp_auto_ads_opt_filtered['data-ad-client'] = $this->tag_id;
+
+		$attributes = '';
+		foreach ( $amp_auto_ads_opt_filtered as $amp_auto_ads_opt_key => $amp_auto_ads_opt_value ) {
+			$attributes .= sprintf( ' %s="%s"', $amp_auto_ads_opt_key, esc_attr( $amp_auto_ads_opt_value ) );
+		}
```

Comment:
We also need to escape attribute names to make sure nobody can't add malicious attributes using our filter. Furthermore, I think we need to move the `data-` prefix from the array keys to the sprintf template to make sure that only `data-*` attributes can be added with our filter:

```php
$amp_auto_ads_opt = array(
    'ad-client' => $this->tag_id,
);

$amp_auto_ads_opt_filtered = apply_filters( 'googlesitekit_amp_auto_ads_attributes', $amp_auto_ads_opt, $this->tag_id );

...

$attributes = '';
foreach ( $amp_auto_ads_opt_filtered as $amp_auto_ads_opt_key => $amp_auto_ads_opt_value ) {
    $attributes .= sprintf( ' data-%s="%s"', esc_attr( $amp_auto_ads_opt_key ), esc_attr( $amp_auto_ads_opt_value ) );
}
```

Reply:
Sounds good. Thanks @eugene-manuilov